app/helpers/service_helper.rb

```diff
@@ -0,0 +1,48 @@
+module ServiceHelper
+  SALESFORCE_LISTING_FIELD_NAMES_TO_TRANSLATE = %w[
```

Comment:
So that we don't have to keep two constants in sync, can we just use LISTING_FIELDS_TO_TRANSLATE and append the __c for when this is used?